theano/gpuarray/elemwise.py

```diff
@@ -75,10 +75,11 @@ def make_node(self, *inputs):
             pass
         try:
             support_code = self.scalar_op.c_support_code()
-            if (support_code.strip() != "#define THEANO_MACRO_MOD(x,y) (x % y)" and
-                    support_code.strip() != ""):
+            if "struct" in support_code:
```

Comment:
The thing is that a struct is ok if not really a C++ class.  But I guess this is ok for now.